CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-# Changelog
+## 0.5.3
+
+- Now uses hashes to maintain stable ids.
```

Comment:
nit: how about "Made ids stable in the json format, which makes plain text diffing possible."